Makefile

```diff
@@ -1,9 +1,12 @@
 up: stop build
 	docker-compose up
 
-build:
+build: volumes
 	docker-compose build
 
+volumes:
+	docker volume create --name=grafana-volume
```

Comment:
Just for curiosity, 
why are you building the volume here and not mapping on docker-compose directly?

Reply:
Volume mapping throws the volume to a directory within the repository. I created a shared volume and there may be other projects that will consume the same volume. This facilitates the management of data created by docker, since we can delete this data with the `docker volume prune grafana-volume` command.